packages/frontend/src/components/Game/GameBoard/ResultDisplay/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import useGameContext from 'hooks/useGameContext';
+
+const ResultDisplay = () => {
+  const {
+    state: { winner },
+  } = useGameContext();
+
+  return (
+    <div
+      data-testid="result-display"
+      className="container d-flex justify-content-center mt-5"
+    >
+      <div className="d-flex-column text-center">
+        <h1 className="mb-5">GAME OVER!</h1>
+
+        {winner.length > 1 ? (
+          <h2 className="lh-lg">Its a tie!</h2>
+        ) : (
+          <h2 className={`text-${winner[0].color.className}`}>
+            {`${winner[0].name} wins!`}
+          </h2>
+        )}
```

Comment:
Instead of looking at the winner array length, the game stage could be set to GAME_OVER and the winner would be `null`